backend/routes/stripe.js

```diff
@@ -62,7 +62,13 @@ module.exports = function (router) {
     const shopConfig = getConfig(req.shop.config)
     const web3Pk = shopConfig.web3Pk || networkConfig.web3Pk
 
-    if (!web3Pk || !shopConfig.stripeBackend) {
+    const valid = await stripeUtils.webhookValidation(
+      req.shop,
+      shopConfig,
+      shopConfig.stripeWebhookHost || networkConfig.backendUrl
+    )
+
+    if (!web3Pk || !valid) {
```

Comment:
It would be worth logging the shop id and the reason for the error. To help troubleshooting when a merchant complains his CC payments are not going thru.
Same comment for the route /stripe/check-creds below.